apps/scan/backend/src/electrical_testing/simple_scanner_client.ts

```diff
@@ -30,7 +30,7 @@ export function createSimpleScannerClient(): SimpleScannerClient {
 
     async disconnect() {
       assert(client, 'Scanner client is not connected');
-      (await client.disconnect()).unsafeUnwrap();
+      (await client.exit()).unsafeUnwrap();
```

Comment:
Ooh good find! Out of curiosity, do we face a similar issue anywhere in the real state machine? Using disconnect and not cleaning up pdictl processes?

Reply:
I'm not sure. We don't appear to be calling `exit` in the PDI state machine. Perhaps @jonahkagan knows whether the reconnect logic makes it so the underlying `pdictl` process gets cleaned up properly?

Reply:
IIRC, it turned out that for all of the disconnections we were handling, it was possible to reconnect by simply calling `client.connect()` again instead of exiting and creating a new client. So the `pdictl` process stays alive. Any situation where the `pdictl` process crashes is treated as an unexpected failure, not a disconnection.

But it's probably worth having someone else take a look to confirm. I just checked but a second pair of eyes wouldn't hurt

Reply:
This comment explains some of how the client works in this regard: https://github.com/votingworks/vxsuite/blob/main/libs/pdi-scanner/src/ts/scanner_client.ts#L215-L218

(Though the comment should say 'exit' error code, not 'disconnected' error code)